api/rhsm-subscriptions-api-spec.yaml

```diff
@@ -654,6 +654,17 @@ components:
           schema:
             $ref: "#/components/schemas/Errors"
   schemas:
+    GranularityGenerated:
+      type: string
+      enum: [ DAILY, WEEKLY, MONTHLY, QUARTERLY, YEARLY ]
+    ServiceLevelGenerated:
+      type: string
+      enum: [ "", Premium, Standard, Self-Support, _ANY ]
+      x-enum-varnames: [ UNSPECIFIED, PREMIUM, STANDARD, SELF_SUPPORT, ANY ]
+    UsageGenerated:
+      type: string
+      enum: [ "", Production, Development/Test, Disaster Recovery, _ANY ]
+      x-enum-varnames: [ UNSPECIFIED, PRODUCTION, DEVELOPMENT_TEST, DISASTER_RECOVERY, ANY ]
```

Comment:
Preferences/Suggestions on naming conventions for these?  I had thrown the word "Generated" at the end of them to prevent the namespace collision with the enums in the db.model package...but the side effect is that the docs include the word "Generated", as seen below.

![image](https://user-images.githubusercontent.com/19955562/100783648-0c78d800-33dc-11eb-8e7d-2f1a158fe837.png)

The reason I care about the name collision thing is because it was kind of confusing keeping them straight, and fully qualifying a package lessens readability too because clutter.  On top of that, checkstyle gets mad & requires SuppressWarning more often because of line length.